packages/manager/apps/pci-savings-plan/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useProject } from '@ovhcloud/manager-components';
+import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
+import {
+  usePciBreadcrumb,
+  BreadcrumbItem,
+} from '@/hooks/breadcrumb/useBreadcrumb';
+import appConfig from '@/pci-savings-plan.config';
+
+export interface BreadcrumbProps {
+  customRootLabel?: string;
+  appName?: string;
+  items?: BreadcrumbItem[];
+}
+
+function Breadcrumb({ customRootLabel }: BreadcrumbProps): JSX.Element {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const label = customRootLabel || appConfig.rootLabel;
+
+  const { projectId } = useParams();
+  const { data: project } = useProject();
+
+  if (project) {
+    const breadcrumbPci = usePciBreadcrumb({
+      projectId,
+      appName: 'pci-savings-plan',
+    });
+    return <OsdsBreadcrumb items={breadcrumbPci} />;
+  }
```

Comment:
Can you fix this sonar bug ?
You don't follow one of the hooks rules

Reply:
This implementation comes from the current APP generator.

Reply:
Then it should probably be fixed in both places